docs/source/posts/2021/2021-06-08-week-1-antriksh.rst

```diff
@@ -0,0 +1,27 @@
+Week #1: Welcome to my weekly Blogs!
+====================================
+
+.. post:: June 08 2021
+   :author: Antriksh Misri
+   :tags: google
+   :category: gsoc
+
+Hi everyone! I am **Antriksh Misri**. I am a Pre-Final year student at MIT Pune. This summer, I will be working on **Layout Management** under FURY's UI module as my primary goal. This includes addition of different classes under Layout Management to provide different layouts/arrangements in which the UI elements can be arranged. As my stretch goals, I will be working on a couple of UI components for FURY’s UI module. These 2D and 3D components will be sci-fi like as seen in the movie “**Guardians of The Galaxy**”. My objective for the stretch goals would be to develop these UI components with their respective test and tutorials such that it adds on to the UI module of FURY and doesn’t hinder existing functionalities/performance.
+
+What did I do this week?
+------------------------
+During the community bonding period I got to know the mentors as well as other participants. We had an introductory meeting, in which the rules and code of conduct was explained. Also, my proposal was reviewed and modified slightly. Initially, I had to develop UI elements as my primary goal and I had to work on layout management as my stretch goals but the tasks were switched. Now I have to work on Layout Management as my primary task and develop UI in the stretch goals period. I also started coding before hand to actually make use of this free period. I worked on different PR's which are described below:-
+
+* `Added tests for Layout module <https://github.com/fury-gl/fury/pull/434>`_ : The layout module of FURY didn't had any tests implemented, so I made this PR to add tests for **Layout** & **GridLayout** class.
+* `Complied available classes for Layout Management in different libraries <https://docs.google.com/document/d/1zo981_cyXZUgMDA9QdkVQKAHTuMmKaixDRudkQi4zlc/edit>`_ : In order to decide the behavior and functionality of Layout Management in FURY, I made a document that has all classes available in different libraries to manage layout of UI elements. This document also contains code snippets for these classes
```

Comment:
> <https://docs.google.com/document/d/1zo981_cyXZUgMDA9QdkVQKAHTuMmKaixDRudkQi4zlc/edit>`

Not sure that this document will be always available. You might need to create an rst file for that. 
I need to think about it

Reply:
I think google doc stays up unless you delete it. Regardless, I can make a rst file, where should that file go?

Reply:
ok, let's keep it for now